src/main/kotlin/no/nav/syfo/client/narmesteleder/NarmesteLederClient.kt

```diff
@@ -22,11 +22,13 @@ class NarmesteLederClient(
     private val narmestelederClientId: String,
     private val azureAdV2Client: AzureAdV2Client
 ) {
-    private val sykmeldtAktivNarmesteLederPath = "$narmesteLederBaseUrl$NARMESTELEDER_CURRENT_PATH?orgnummer="
+    private val sykmeldtAktivNarmesteLederPath = "$narmesteLederBaseUrl$CURRENT_NARMESTELEDER_PATH?orgnummer="
+    private val sykmeldtNarmesteLederePath = "$narmesteLederBaseUrl$NARMESTELEDERE_PATH?utvidet=ja"
+    private val ansatteNarmesteLederPath= "$narmesteLederBaseUrl$CURRENT_ANSATTE_PATH"
```

Comment:
Disse variablene synes jeg er forvirrende 😕 
De to første kan hete "sykmeldt..." fordi `CURRENT_NARMESTELEDER_PATH` og `NARMESTELEDERE_PATH` begynner med "/sykmeldt". Men den siste, `CURRENT_ANSATTE_PATH`, begynner på "/leder" så da er det rart å kalle variabelen "ansatte".
Hvis navnene er basert på hva som returneres fra pathen, burde de to første begynne på "leder(e)" 🤷‍♂️